dipy/nn/tests/test_utils.py

```diff
@@ -31,10 +31,13 @@ def test_transform(rng=None):
         )
         temp2 = recover_img(
             temp2,
-            new_affine,
-            ori_shape,
+            affine,
+            mid_shape,
             temp.shape,
-            init_shape=(32, 32, 32),
-            voxsize=np.ones(3) * 2,
+            offset_array,
+            np.ones(3) * 2,
+            scale,
+            crop_vs,
+            pad_vs,
```

Comment:
missing all keyword arguments

Reply:
finally make sense if all became mandatory

Reply:
Yes it is mandatory indeed. Sorry for the confusion